yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -202,18 +272,25 @@ export class AvmPersistableStateManager {
   public writeL1Message(recipient: EthAddress | Fr, content: Fr) {
     this.log.debug(`L1Messages(${recipient}) += ${content}.`);
     const recipientAddress = recipient instanceof EthAddress ? recipient : EthAddress.fromField(recipient);
-    this.newL1Messages.push(new L2ToL1Message(recipientAddress, content));
+    const message = new L2ToL1Message(recipientAddress, content);
+    this.newL1Messages.push(message);
+
+    // TRANSITIONAL: This should be removed once the AVM is fully operational and the public executor is gone.
+    this.transitionalExecutionResult.newL2ToL1Messages.push(message);
   }
 
   public writeLog(contractAddress: Fr, event: Fr, log: Fr[]) {
     this.log.debug(`UnencryptedL2Log(${contractAddress}) += event ${event} with ${log.length} fields.`);
-    const L2log = new UnencryptedL2Log(
+    const ulog = new UnencryptedL2Log(
```

Comment:
I just didn't like the var name starting with a capital letter, and in my PR (before merging in master) I had given it the name ulog before realizing it was also assigned a name in master. I don't feel strongly about the name!